extensions/ffmpeg/src/main/jni/ffmpeg_jni.cc

```diff
@@ -275,7 +277,7 @@ int decodePacket(AVCodecContext *context, AVPacket *packet,
       av_opt_set_int(resampleContext, "in_sample_rate", sampleRate, 0);
       av_opt_set_int(resampleContext, "out_sample_rate", sampleRate, 0);
       av_opt_set_int(resampleContext, "in_sample_fmt", sampleFormat, 0);
-      av_opt_set_int(resampleContext, "out_sample_fmt", OUTPUT_FORMAT, 0);
+      av_opt_set_int(resampleContext, "out_sample_fmt", use32BitFloatOutput ? OUTPUT_FORMAT_32FLOAT : OUTPUT_FORMAT, 0);
```

Comment:
I believe the answer is yes.  That is something that is probably an easy test I can do later.